airflow_prometheus_exporter/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 __author__ = """Robinhood Markets, Inc."""
 __email__ = "open-source@robinhood.com"
-__version__ = "__version__ = '1.1.2j'"
+__version__ = "__version__ = '1.1.3j'"
```

Comment:
No. Please don't edit this file.

This might cause some issues, and because of that some projects might fail. For example, lets assume you fork Airflow. 

There might be checks on different libraries to do different things based on the Airflow version. So if we change the patch version this might break the logic, because the versions won't match